exporter/signalfxexporter/exporter.go

```diff
@@ -167,6 +179,17 @@ func newEventExporter(config *Config, logger *zap.Logger) (*signalfxExporter, er
 				// TODO: What other settings of http.Client to expose via config?
```

Comment:
Do you mind removing this comment since this is a major step for the item?

Reply:
Sure, I will do that.